### PR TITLE
update tests

### PR DESCRIPTION
**Pull request by ryanlong1004**
Tests the 'print job card' functionality.

### CODE REVIEW

File: tests/test_scheduler.py [hunk 1]
```diff
@@ -468,5 +468,7 @@ def test_string_output():
     }
 
     js = JobScheduler.get_scheduler(props)
-    actual = js.job_card.content()
+    jc = js.job_card
+    actual = str(jc)
+    print(jc)
```
Comment (christinaholtNOAA) [inline tests/test_scheduler.py:473]: I assume that we want to expect that `__str__` is actually returning a string, and don't want to hide that potential oversight (an inadvertent change to the `__str__` method) in the `str()` on line 472. Could it just be:
```
actual = js.job_card
assert actual == expected
```

Also, would it be okay to remove the `print()`?

**Review by christinaholtNOAA — CHANGES_REQUESTED** [review]
Looks good.

**Comment by aerorahul** [comment]
waiting on resolution from @christinaholtNOAA's comment.
And then we can close/merge this.

File: tests/test_scheduler.py (hunk 1 above)
Comment (aerorahul) [inline tests/test_scheduler.py:473]:
```suggestion
```

**Review by aerorahul — APPROVED** [review]
looks good.

**Comment by ryanlong1004** [comment]
@christinaholtNOAA bump

**Pull request merged**